src/AppShed/Extensions/SpreadsheetBundle/Service/GeoService.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace AppShed\Extensions\SpreadsheetBundle\Service;
+
+use GuzzleHttp\Client;
+
+/**
+ * Description of GeoService
+ *
+ * @author Victor
+ */
+class GeoService {
+
+    public function getGeo($address)
+    {
+        $guzzle = new Client();
+
+        $address = urlencode($address);
+
+        $url = "http://maps.google.com/maps/api/geocode/json?sensor=false&address={$address}";
+
+        $res = $guzzle->get($url);
```

Comment:
Basically you want to make a big `try` `catch` around this whole function, particularly this bit. You should be trying to catch `GuzzleHttp\Exception\RequestException` and return something appropriate. Probably `false`.